layers/+email/mu4e/packages.el

```diff
@@ -39,9 +39,28 @@
                 'ivy-completing-read
               'helm-completing-read))
 
+      (when mu4e-async
+        (require 'smtpmail-async)
+        (mu4e/set-send-function 'async-smtpmail-send-it))
+
+      (when (fboundp 'imagemagick-register-types)
+        (imagemagick-register-types))
+
+      (let ((dir "~/Downloads"))
```

Comment:
Ohhh...I guess I could just directly set `mu4e-attachment-dir` myself :smile: 
This makes sense as a default though :+1: